test/factory/album.js

```diff
@@ -0,0 +1,7 @@
+const { factory } = require('factory-girl');
+const faker = require('faker');
+const models = require('../../app/models/index');
+
+factory.define('album', models.album, {
+  title: faker.name.jobTitle
```

Comment:
fyi, `factory-girl` has [chance.js](https://chancejs.com/) included, and you can access it via `factory.chance`, so there is no need to use another package like `faker`.  If you like `faker` there is no need to change it tho, it looks preety good.